### ISSUE TITLE
RCS support is broken

### ISSUE THREAD

**Issue by manuel-192** [MightyCreak/diffuse#28]
Diffuse no more shows differences for RCS files.

**Comment by uffejakobsen** [comment]
I guess that this is a duplicate of issue #26 

And I guess that my pull request #27 will also fix your problem with RCS (I only tested my fix with GIT)

So now we are just waiting for the repo owner to integrate the pull requests....   :-)

**Comment by uffejakobsen** [comment]
BTW: if you use ArchLinux - they have integrated my patch into their package of diffuse

**Comment by MightyCreak** [comment]
I merged your PR @uffejakobsen ;)

**Comment by manuel-192** [comment]
Thanks!

**Comment by manuel-192** [comment]
Hello again!
I still have the same RCS problem, it doesn't show the diffs. Do I have the correct version or am I missing something?
Command 'git describe --long --tags' shows
v0.5.0-alpha.7-8-gb1f7036

**Comment by MightyCreak** [comment]
What's your latest commit hash?

**Comment by MightyCreak** [comment]
Also, could you give me an example of RCS file please? I don't know this extension :confused:

**Comment by manuel-192** [comment]
I guess the hash is: b1f70363ab938e2f6aaf0f7c109db06bf323c7cf.

File 'foobar': http://ix.io/2fYx
File 'RCS/foobar,v': http://ix.io/2fYy

So the version control file is under the RCS folder. Command
diffuse foobar
does not show the diffs at all as it should.

**Comment by MightyCreak** [comment]
I found the bug :tada:
You can see the commit description here: c5b0dc1

TL;DR: it was actually a bug from the original repo, introduced by r420 (a revision committed after the 0.4.8 release).

**Comment by manuel-192** [comment]
Thanks! I'll try it here.
